app/javascript/dashboard/routes/dashboard/settings/automation/AddAutomationRule.vue

```diff
@@ -259,7 +260,35 @@ export default {
       return false;
     },
   },
+  mounted() {
+    const customAttributesRaw = this.$store.getters['attributes/getAttributes'];
+    const customAttributeTypes = customAttributesRaw.map(attr => {
+      return {
+        key: attr.attribute_key,
+        name: attr.attribute_display_name,
+        inputType: this.customAttributeInputType(attr.attribute_display_type),
+        filterOperators: this.getOperatorTypes(attr.attribute_display_type),
+      };
+    });
+    AUTOMATIONS.message_created.conditions.push(...customAttributeTypes);
+    AUTOMATIONS.conversation_created.conditions.push(...customAttributeTypes);
+    AUTOMATIONS.conversation_updated.conditions.push(...customAttributeTypes);
```

Comment:
@fayazara These logic seems to be repeated in the edit AutomationRule as well. Can we create a mixin or use same component in both places?